docs/4.0/utilities/display.md

```diff
@@ -6,20 +6,34 @@ group: utilities
 toc: true
 ---
 
-## Common `display` values
+## How it Works
 
-The [`display` property](https://developer.mozilla.org/en-US/docs/Web/CSS/display) accepts a handful of values and we support many of them with utility classes. We purposefully don't provide every value as a utility, so here's what we support:
+Change the `display` value of elements with our responsive-friendly display utility classes.
 
-- `.d-none`
-- `.d-inline`
-- `.d-inline-block`
-- `.d-block`
-- `.d-table`
-- `.d-table-cell`
-- `.d-flex`
-- `.d-inline-flex`
+The [`display`](https://developer.mozilla.org/en-US/docs/Web/CSS/display) property accepts lots of values, and we support many of them with utility classes. We purposefully don't provide them all.
 
-Put them to use by applying any of the classes to an element of your choice. For example, here's how you could use the inline, block, or inline-block utilities (the same applies to the other classes).
+## Notation
+
+The classes are named using the format 
```

Comment:
Let's use the same approach from the Spacing docs. Also, always include a line between paragraphs and always use punctuation.

> Display utility classes that apply to all breakpoints, from `xs` to `xl`, have no breakpoint abbreviation in them. This is because those classes are applied from `min-width: 0` and up, and thus are not bound by a media query. The remaining breakpoints, however, do include a breakpoint abbreviation.
>
> As such, the classes are named using the format `.d-{value}` for `xs` and `.d-{breakpoint}-{value}`  for `sm`, `md`, `lg`, and `xl`.

Markdown for copy-pasting:

```
Display utility classes that apply to all breakpoints, from `xs` to `xl`, have no breakpoint abbreviation in them. This is because those classes are applied from `min-width: 0` and up, and thus are not bound by a media query. The remaining breakpoints, however, do include a breakpoint abbreviation.

As such, the classes are named using the format `.d-{value}` for `xs` and `.d-{breakpoint}-{value}`  for `sm`, `md`, `lg`, and `xl`.
```

Reply:
still like mine, but compromised with bullet points to make it stand out for newbies.